CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to Contribute
+
+First off, thank you for contributing! We're excited to collaborate with you! 🎉
+
+The following is a set of guidelines for the many ways you can join our collective effort.
+
+Before anything else, please take a moment to read our [Code of Conduct](CODE-OF-CONDUCT.md). We expect all participants, from full-timers to occasional tinkerers, to uphold it.
```

Comment:
I believe we need underscores here for the code of conduct filename (`CODE_OF_CONDUCT.md`).

Reply:
I went ahead and made this update myself.